packages/features/ee/dsync/lib/handleGroupEvents.ts

```diff
@@ -185,7 +185,7 @@ const handleGroupEvents = async (event: DirectorySyncEvent, organizationId: numb
     });
 
     // Send emails to new members
-    const newMembers = users.filter((user) => !user.teams.find((team) => team.id === group.teamId));
+    const newMembers = users.filter((user) => !user.teams.find((team) => team.teamId === group.teamId));
```

Comment:
We should rename _team_ to _membership_.

I have been bitten by a similar bug too 😂